assets/src/pages/webrtcInternals/Details.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { SyntheticEvent, useCallback, useEffect, useState } from "react";
+
+type DetailsProps = {
+  summaryText: string;
+  children: React.ReactNode;
+  isOpen: boolean;
+  toggle: () => void;
+};
+
+const Details = ({ summaryText, children, isOpen, toggle }: DetailsProps) => {
+  const [isMounted, setMount] = useState<boolean>(false);
+
+  const unbubbledToggle = useCallback(
+    (e: SyntheticEvent) => {
+      e.stopPropagation();
+      isMounted && toggle();
+    },
+    [isMounted, toggle]
+  );
+
+  useEffect(() => {
+    setMount(true);
+  }, []);
+
+  return (
+    <details onToggle={unbubbledToggle} open={isOpen}>
+      <summary>{summaryText}</summary>
+      {children}
+    </details>
+  );
+};
```

Comment:
There were several problems with the `details` HTML element like:
 1. The mount happened on every rerender.  -> Solution: persisting it
 2. The element seemed to double toggle and based on [a solution from SO](https://stackoverflow.com/questions/58942600/react-html-details-toggles-uncontrollably-when-starts-open) I thought it will be a good idea to check if the element is mounted or not.
 3. The details element bubbles so using it recursively has lead to some strange behaviour.

I stumbled upon those problem in the order above so  it may be so that the are to many not standard solutions and i.e. leaving only the *debubbling* part is sufficient. I'll give it a try.